src/map_science/core.cljs

```diff
@@ -70,10 +70,15 @@
                          :width 5
                          :background-color (if isDragged "#548BF4" "#CCC")}})))
 
-
+;; Creating own React hooks
+(defn use-local-state [tkey initial]
+  (let [[v, u] (hooks/use-state (if (js/localStorage.getItem tkey) (js/JSON.parse (js/localStorage.getItem tkey)) initial))]
```

Comment:
@f1729  I'm looking for a way to simplify it. Do you know any?

Reply:
Yes, first, the function that verify if exist some value in localStorage should be in another function, functional programming promote the use of little function that do little tasks.

Reply:
I could be more descriptive in -> let [[v, u]

Reply:
The rest LGTM

Thank you! I will add you in the acknowledgement :+1: